package/yast2-geo-cluster.spec

```diff
@@ -17,13 +17,12 @@
 
 
 Name:           yast2-geo-cluster
-Version:        3.1.14
+Version:        3.2.1
```

Comment:
We usually start with `0` when raising the minor version, i.e. it should be `3.2.0` in this case.